src/app/features/todos/todo.service.ts

```diff
@@ -19,14 +19,12 @@ export class TodoService {
     const nextTodo = await this.createNextRepetition(todo);
 
     todo.completed = {
-      completedBy: this.currentUser.value.getId(),
+      completedBy: this.currentUser.value.getId(true),
       completedAt: new Date(),
       nextRepetition: nextTodo?.getId(true),
     };
 
     await this.entityMapper.save(todo);
-
-    // TODO: user block instead of id to display in template
```

Comment:
Currently, I use `.toString()` because the display block component looks really bad. @sleidig do you want to have a look at it or are we fine with just showing the name?

Reply:
agreed